playbooks/responses/how-it-works.md

```diff
@@ -0,0 +1,6 @@
+Open source is a significantly volunteer-driven and collaborative system of production. Because of this structure, time and energy to get things done is the scarcest of resources in open source projects. We want Electron to be the best that it can be, but we also have to be realistic about what the team maintaining Electron can achieve. This means that:
```

Comment:
```suggestion
Open source is largely volunteer-driven and collaborative. Because of this, time and energy to get things done are often the scarcest resources. We want Electron to be the best that it can be, but we also have to be realistic about what the team maintaining Electron can achieve. This means that:
```

Minor edits for the PR's goal of making the warning shorter